play/server/src/main/scala/endpoints/play/server/Endpoints.scala

```diff
@@ -168,14 +201,24 @@ trait Endpoints extends algebra.Endpoints with Urls with Methods {
   type Response[A] = A => Result
 
   /** A successful HTTP response (status code 200) with no entity */
-  lazy val emptyResponse: Response[Unit] = _ => Results.Ok
+  def emptyResponse(docs: Documentation): Response[Unit] = _emptyResponse
+  private lazy val _emptyResponse: Response[Unit] = _ => Results.Ok
 
   /** A successful HTTP response (status code 200) with string entity */
-  lazy val textResponse: Response[String] = x => Results.Ok(x)
+  def textResponse(docs: Documentation): Response[String] = _textResponse
+  private lazy val _textResponse: Response[String] = x => Results.Ok(x)
```

Comment:
Here again I would just inline `_emptyResponse` and `_textResponse`.

Reply:
done